### PR TITLE
Ingresses

### PR DESCRIPTION
**Pull request by akucharska**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: core-ui/src/components/Predefined/List/Ingresses.list.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+export const IngressesList = ({ DefaultRenderer, ...otherParams }) => {
+  const { t } = useTranslation();
+
+  const getLoadBalancer = service => {
+    if (service.status.loadBalancer?.ingress) {
+      return service.status.loadBalancer?.ingress
+        .map(endpoint => endpoint.ip || endpoint.hostname)
+        .join(', ');
+    } else {
+      return '-';
```
Comment (Wawrzyn321) [inline core-ui/src/components/Predefined/List/Ingresses.list.js:15]: Please use EMPTY_TEXT_PLACEHOLDER.

File: core/src/i18n/en.yaml [hunk 2]
```diff
@@ -981,6 +981,28 @@ git-repositories:
...
+ingresses:
+  description: <0>Ingress</0> exposes HTTP and HTTPS routes from outside the cluster to services within the cluster. Traffic routing is controlled by rules defined on the Ingress resource.
+  labels:
+    apiGroup: ApiGroup
+    backend: Backend
+    default-backend: Default backend
+    host: Host
+    ingress-class-name: Ingress class name
+    kind: Kind
+    load-balancers: LoadBalancers
```
Comment (Wawrzyn321) [inline core/src/i18n/en.yaml:993]:
```suggestion
    load-balancers: Load Balancers
```
...?

File: tests/tests/test-ingresses.spec.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+      .find('[role="presentation"],[class="view-lines"]')
+      .first()
+      .type(
+        '{selectall}{backspace}{selectall}{backspace}{selectall}{backspace}',
+      );
+
+    cy.wrap(loadRandomIngress(NAME, Cypress.env('NAMESPACE_NAME'))).then(
+      INGRESS_CONFIG => {
+        const INGRESS = JSON.stringify(INGRESS_CONFIG);
+        console.log('INGRESS', INGRESS);
```
Comment (Wawrzyn321) [inline tests/tests/test-ingresses.spec.js:48]:
```suggestion
```

File: tests/tests/test-ingresses.spec.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+        cy.getIframeBody()
+          .find('[role="presentation"],[class="view-lines"]')
+          .first()
+          .click()
+          .type(INGRESS, { parseSpecialCharSequences: false });
```
Comment (Wawrzyn321) [inline tests/tests/test-ingresses.spec.js:53]:
```suggestion
          .paste({
            pastePayload: INGRESS,
          });
```

**Review by Wawrzyn321 — CHANGES_REQUESTED** [review]
This translation is missing
image

File: core-ui/src/components/Predefined/Details/Ingress/RowComponent.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { FormItem, FormLabel } from 'fundamental-react';
+import './RowComponent.scss';
+
+export const RowComponent = ({ name, value }) => (
```
Comment (Wawrzyn321) [inline core-ui/src/components/Predefined/Details/Ingress/RowComponent.js:5]: I believe we can use `LayoutPanelRow` for that (well, it has no ':'). Besides, `item-wrapper` class is not defined anywhere - AFAIK it has been renamed to `break-word` (from root index.scss).

**Review by Wawrzyn321 — DISMISSED** [review]

File: core/src/i18n/en.yaml [hunk 6]
```diff
@@ -981,6 +981,28 @@ git-repositories:
     auth: 'Basic: token or password / SSH key: authentication key / Public: no authentication.'
     secret: The name of the Secret with credentials used by the Function Controller to authenticate to the Git repository in order to fetch the Function's source code and dependencies.
     url: URL must be a valid Git repository address that starts with the "http(s)", "git", or "ssh" prefix.
+ingresses:
+  description: <0>Ingress</0> exposes HTTP and HTTPS routes from outside the cluster to services within the cluster. Traffic routing is controlled by rules defined on the Ingress resource.
```
Comment (grego952) [inline core/src/i18n/en.yaml:985]:
```suggestion
  description: <0>Ingress</0> describes the routing rules for traffic to services within a cluster. The routing is managed by the Ingress resource.
```

**Review by Wawrzyn321 — APPROVED** [review]

**Review by Wawrzyn321 — DISMISSED** [review]

File: core/src/i18n/en.yaml [hunk 7]
```diff
@@ -1002,6 +1002,28 @@ git-repositories:
     none: Authentication not required.
     secret: The name of the Secret with credentials used by the Function Controller to authenticate to the Git repository in order to fetch the Function's source code and dependencies.
     url: URL must be a valid Git repository address that starts with the "http(s)", "git", or "ssh" prefix.
+ingresses:
+  description: <0>Ingress</0> describes the routing rules for traffic to services within a cluster. The routing is managed by the Ingress resource.
+  labels:
+    apiGroup: ApiGroup
```
Comment (majakurcius) [inline core/src/i18n/en.yaml:1008]:
```suggestion
    apiGroup: APIGroup
```
`API` is an acronym for `Application Programming Interface`, unless there's a reason why it's spelt this way here that I'm not aware of.

**Review by majakurcius — DISMISSED** [review]

**Review by majakurcius — APPROVED** [review]

**Review by Wawrzyn321 — APPROVED** [review]

**Pull request merged**